packages/web/src/pages/upload-page/forms/EditTrackFormForUpload.tsx

```diff
@@ -0,0 +1,65 @@
+import { useCallback, useMemo } from 'react'
+
+import moment from 'moment'
+
+import { EditTrackForm } from 'components/edit-track/EditTrackForm'
+// TODO: maybe move this to a shared location
```

Comment:
do we want a ticket for this TODO? nbd tho can ignore.

Reply:
I think we did this. This was referring to the defaultHiddenFields below which are now in `components/edit`. Gonna update the imports though to be `'components/edit/...'`